.circleci/config.yml

```diff
@@ -54,17 +91,11 @@ jobs:
             make docker.all generate_yaml
       - run: bin/testEnvRootMinikube.sh wait
       - run: docker images
-      - run: make test/minikube/auth/e2e_simple
```

Comment:
hey I had fixed this test to be running with auth (it needs to) - this PR reverted my fix